src/mono/netcore/sample/HelloWorld/Makefile

```diff
@@ -0,0 +1,24 @@
+TOP=../../../../../
+DOTNET:=$(TOP)./dotnet.sh
+DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
+
+MONO_CONFIG=Release
+MONO_ARCH=x64
+
+OS := $(shell uname -o)
+ifeq ($(OS),Darwin)
+	TARGET_OS=osx
+else
+	TARGET_OS=linux
+endif
+
+publish:
+	$(DOTNET) publish -c $(MONO_CONFIG) -r $(TARGET_OS)-$(MONO_ARCH)
+
+run: publish
+	COMPlus_DebugWriteToStdErr=1 \
+	MONO_ENV_OPTIONS="--llvm" \
```

Comment:
Why is this `--llvm` here? can I run hello world with another EE?

Reply:
Just a placeholder for custom args 🙂 if llvm is not built - it'll print a warning but the app will still work.